### DISCUSSION TITLE
Exclude own WAN IP from statistics

### DISCUSSION THREAD

**Discussion by DonniDoc** [tobychui/zoraxy#389] [Ideas]
First of all, thanks for the great tool! For me, Zoraxy is the best proxy manager.

I'm looking for a solution for the following: Since I've been running Nextcloud AIO behind Zoraxy, the statistics in Zoraxy have become more or less useless. Nextcloud seems to ping the defined domain (and thus WAN address) every 10 seconds. This means that in the pie charts you see my country, my operating system, etc. as almost the whole pie. Is there a way to exclude your own WAN IP from the statistics? And it must be the determined WAN IP. I don't have a fixed IP...

Thanks to all!

**Comment by tobychui** [comment]
Probably not possible with Zoraxy. Your request is too specific to your use case.
